docs/index.asciidoc

```diff
@@ -133,6 +148,19 @@ fingerprint computation. If `false` and at least one source field is
 given, the target field will be an array with fingerprints of the 
 source fields given.
 
+[id="plugins-{type}s-{plugin}-ecs_compatibility"]
+===== `ecs_compatibility`
+
+* Value type is <<string,string>>
+* Supported values are:
+** `disabled`: unstructured data added at root level
+** `v1`: uses `[event][hash]` fields that are compatible with Elastic Common Schema
+
+Controls this plugin's compatibility with the
+{ecs-ref}[Elastic Common Schema (ECS)].
+See <<plugins-{type}s-{plugin}-ecs_metadata>> for detailed information.
+The value of this setting affects the _default_ value of <<plugins-{type}s-{plugin}-???>>
```

Comment:
`<<plugins-{type}s-{plugin}-???>>` will break the docs because it can't be resolved to anything. I wasn't sure what you wanted to link back to, and so I didn't offer a suggestion.

Reply:
good catch. A copy and paste error on my side.